Source/WebInspectorUI/UserInterface/Views/OpenResourceDialog.js

```diff
@@ -396,6 +399,9 @@ WI.OpenResourceDialog = class OpenResourceDialog extends WI.Dialog
         const suppressFilterUpdate = true;
 
         let targetData = WI.debuggerManager.dataForTarget(target);
+        if (!targetData)
+            return;
```

Comment:
this prevents the `target.extraScriptCollection` below.  do we want that?  or is that still potentially useful?
```js
let targetData = WI.debuggerManager.dataForTarget(target);
for (let script of (targetData?.scripts || [])) {
```